src/components/charts/PercentageStackedBarChart.jsx

```diff
@@ -0,0 +1,76 @@
+import { BarChart, Bar, Legend, Tooltip, XAxis, YAxis, CartesianGrid,
+} from 'recharts';
+import PropTypes from 'prop-types';
+import React from 'react';
+import _ from 'underscore';
+import { localTheme } from '../../localconf';
```

Comment:
is there a way to pass this as a prop vs. importing? I just feel like the components shouldn't be reaching into the config file